### ISSUE TITLE
Error loading data

### ISSUE THREAD

**Issue by Bay0u** [grishka/Houseclub#243]
**Describe the bug**
So apparently u can see rooms from outside , trying to enter or open followers or following pages it shows "error loading data" message .

**To Reproduce**
1. Enter any room 
2. Click on followers
3. Click on following

**Expected behavior**
I hope app fixed and be back .

**Screenshots**
![Screenshot_20210304-200402_Houseclub](https://user-images.githubusercontent.com/79049309/110016626-8e530700-7d25-11eb-9184-cc1a02a48b84.jpg)
![Screenshot_20210304-200435_Houseclub](https://user-images.githubusercontent.com/79049309/110016645-9317bb00-7d25-11eb-9023-881ecb1b159c.jpg)
![Screenshot_20210304-200440_Houseclub](https://user-images.githubusercontent.com/79049309/110016660-97dc6f00-7d25-11eb-8d6b-85f0541216c6.jpg)

**Smartphone (please complete the following information):**
 - Device: samsung galaxy s9 plus
 - OS: [Android 10]
 - Version [e.g. 1.0.8]

**Additional context**
Thats it.

**Comment by oceanchinatownredpie** [comment]
I experienced the same!

**Comment by caasn** [comment]
Have a look at notifications, maybe you're suspended, because that was what I can see after being suspended.

**Comment by ryogto** [comment]
I saw the same notification after my account was suspended.

**Comment by GhanemGh** [comment]
same here.. "error loading data" for every attempt to enter any room ...

**Comment by Bay0u** [comment]
> Have a look at notifications, maybe you're suspended, because that was what I can see after being suspended.

Yeah you are sadly right i got suspended i don't know why whatever thanks for mentioning.

**Comment by armukrith** [comment]
got same problem. Hope they will fix it soon.

**Comment by caasn** [comment]
Almost one week after suspension I can't read notifications any more. Does anyone know if the reactivate an account after suspending it for spamming?

**Comment by sukhbold** [comment]
Same here, facing error loading data & got suspended. Please fix these bugs ASAP.

**Comment by caasn** [comment]
Being kicked out is not a bug.

**Comment by Ibrahim-T1** [comment]
> Have a look at notifications, maybe you're suspended, because that was what I can see after being suspended.

Same problem happen with me with all rooms

**Comment by Ibrahim-T1** [comment]
![Screenshot_2021-03-12-12-05-08-400_ru turbominer houseclub](https://user-images.githubusercontent.com/80221757/110925569-0922b000-832c-11eb-90c3-1e6eaf15b773.jpg)
I have same problem any help???

**Comment by loaff** [comment]
Can somone fix that please?

**Comment by shatnawiab** [comment]
Guys, U can use VPN for any country N its probably gonna work, But unfortunately U'll have to turn on the VPN whenever U want to use the app, Otherwise the app will bug and U'll face the error again, Hope I helped❤️

**Comment by illtellyoulater** [comment]
Same here, see https://github.com/grishka/Houseclub/issues/298

**Comment by illtellyoulater** [comment]
> Being kicked out is not a bug.

@caasn are you saying this error shows up when someone has been kicked out? Are you sure about this? 
I just registered and I'm seeing this error whenever I try to open a room or my profile page. So being kicked out for what, if I never got to join any room in the first place?
